analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -85,6 +85,15 @@ private fun SpdxDocument.projectPackage(): SpdxPackage? {
     return packages.singleOrNull { it.packageFilename.isEmpty() }
 }
 
+/**
+ * Returns true if the [SpdxDocument] packages contain a package with
+ * the given spdxId
+ */
+private fun SpdxDocument.containsPackage(spdxPackageId: String): Boolean {
+    val spdxPackage = packages.singleOrNull { it.spdxId == spdxPackageId }
+    return spdxPackage != null
```

Comment:
So, if there were *multiple* `spdxPackageId` in the document, this would return `false`, which seems counter-intuitve. It probably should neither return `false` nor `true` in this case, but fail.

Reply:
Actually, I believe we should add a sanity check to `SpdxDocument` itself to ensure that we have no duplicate package IDs in there. I'll look into that.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/3626.

Reply:
Just as a reference, you have done this in https://github.com/oss-review-toolkit/ort/pull/3626

Reply:
Heh, I was quicker 😋